release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -71,17 +71,32 @@ For more information, see xref:../installing/cluster-capabilities.adoc#cluster-c
 [id="ocp-4-16-web-console_{context}"]
 === Web console
 
+[id="ocp-4-16-patternfly4-deprecated"]
+==== Patternfly 4 is now deprecated with 4.16
+With this release, Patternfly 4 and React Router 5 is deprecated in the web console. All plugins should migrate to Patternfly 5 and React Router 6 as soon as possible.
+
 [id="ocp-4-16-administrator-perspective_{context}"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
-//Are there any for 4.16
+
+* A Google Cloud Platform (GCP) token authorization was added to the *Infrastructure features* filter in the OperatorHub.
```

Comment:
Shall we be more explicit here? The filter text is `Auth Token GCP` 
![Screenshot 2024-06-13 at 9 37 00 AM](https://github.com/openshift/openshift-docs/assets/12692381/d9bebf8c-a335-4013-b238-f99aabd1f944)

Reply:
we also have a new `Configurable TLS ciphers` filter added from 4.16, the PR is https://github.com/openshift/console/pull/13555 
Shall we highlight in release note?